### PR TITLE
LOJ-1009 (Back to Underworld) Editorial in English

### PR DESCRIPTION
**Pull request by sakib3201**
Added the English tutorial for LOJ-1009 with links to resources and solution code. Please check the updates .

### CODE REVIEW

File: 1009/en.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# LOJ-1009: Back to Underworld
+---
+**What the problem wants :**You will be given 'n' number of fights between lykans and vampires. The fight will be given as two numbers 'u' and 'v'. And we have to find the maximum number of lykan or vampire. We will need to output the maximum number of members, not which faction they belong to.
```
Comment (showrav-ansary) [inline 1009/en.md:3]: Please create a space between `**` and `You will be.......`. As you can too see, the texts have not been emphasized/bolden.

File: 1009/en.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+So these are the steps we will take to solve the problem:
+  1. Create an adjacency matrix for the graph.
+  2. Use a modified BFS on each node to mark them as either a lykan or vampire node.
+  3. Find the sum of each type of node, compare two sums and the highest one will be our answer.   
+
+**Technical Notes :** The input dataset for this problem is huge. This means using any inefficient method for taking inputs will result in exceeding the time limit. So, make sure to use very fast implementation of input and output methods of which ever language you are using. For example:
+in C++ using "cin" & "cout" might result in time limit exceeding even if the entire implementation is algorithmically correct . This can be avoided by using "scanf" and "printf".
+
+**Additional Resources:**If you don't know what BFS, adjacency matrix or graphs are you can check these awesome resources:
```
Comment (showrav-ansary) [inline 1009/en.md:16]: Please create a space between `**` and `If you don't...........`. As you can too see, the texts have not been emphasized/made bold.

File: 1009/en.md (hunk 1 above)
Comment (showrav-ansary) [inline 1009/en.md:3]: Please create a space between `**` and `You will be.......`. As you can too see, the texts have not been emphasized/made bold.

**Comment by showrav-ansary** [comment]
@sakib3201 Hello dear contributor, please address the above mentioned issues and I will merge them. Please do try to include implementation examples like the states and changes so that it becomes more apparent from next time. Expecting you will include states and changes included editorial next time. Thank you for your continuous contributions ❤️  Keep up the good work ❤️ 

Also, as you too can see, your previous commits are still showing up and so are the modifications. To avoid this, you can create a branch each time you want to contribute a tutorial or after a successful merge, delete your fork and again fork the repo, that way your fork will be up to date as well as your previous commits that are already merged won't show up still.

**Comment by sakib3201** [comment]
Thanks for bringing the problems to my attention. I have fixed the issues on emphasizing the text. I believe adding some illustration of state & changes would greatly increase the clarity of this editorial since it's a graph problem. I am hoping that this editorial is acceptable for now. 
I am sure to upload some illustrations within a week.

**Comment by showrav-ansary** [comment]
@sakib3201 Dear contributor, thank you for your continuous contributions ❤️  Merged 💯 

Yes, you should have the image uploaded within the same folder of the editorial. We suggest relative linking, since the photo you are linking through an url may/may not change and the editorial may get ruined for that. So download/design the photo and keep it in the folder of that particular problem. `html` tags can be used to relative linking but we ask you to do it with `markdown` syntax only.

**Pull request merged**